plugin.video.puls4/default.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import urllib,urllib2,re,xbmcplugin,xbmcgui,sys,xbmcaddon,base64,socket,datetime,time,os,os.path,urlparse,json
+import CommonFunctions as common
+
+version = "0.2.3"
+plugin = "Puls 4 -" + version
+author = "sofaking"
+
+#init
+common.plugin = plugin
+settings = xbmcaddon.Addon(id='plugin.video.puls4') 
```

Comment:
ìd`is not needed here: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcaddon.html